tests/test_ui_mpldrawer.py

```diff
@@ -224,36 +225,41 @@ def test_layered_circuit():
 
 
 def test_fused_gates():
-    c = Circuit(3)
-    c.add(gates.H(0))
-    c.add(gates.X(0))
-    c.add(gates.H(0))
-    c.add(gates.X(1))
-    c.add(gates.H(1))
-    _, fig = plot_circuit(c.fuse(), scale=0.8, cluster_gates=False)
+    circuit = Circuit(3)
+    circuit.add(gates.H(0))
+    circuit.add(gates.X(0))
+    circuit.add(gates.H(0))
+    circuit.add(gates.X(1))
+    circuit.add(gates.H(1))
+    _, fig = plot_circuit(circuit.fuse(), scale=0.8, cluster_gates=False)
     assert match_figure_image(fig, BASEPATH + "/test_fused_gates.npy") == True
 
 
 def test_fuse_cluster():
     """Test for clustering gates"""
-    c = Circuit(2)
-    c.add(gates.X(0))
-    c.add(gates.X(0))
-    c.add(gates.X(1))
-    c.add(gates.M(qubit) for qubit in range(2))
-    _, fig = plot_circuit(c.fuse())
+    circuit = Circuit(2)
+    circuit.add(gates.X(0))
+    circuit.add(gates.X(0))
+    circuit.add(gates.X(1))
+    circuit.add(gates.M(qubit) for qubit in range(2))
+    _, fig = plot_circuit(circuit.fuse())
     assert match_figure_image(fig, BASEPATH + "/test_fuse_cluster.npy") == True
 
 
 def test_plot_unitaries():
     """Test for plotting unitaries"""
-    c = Circuit(6)
-    c.add(gates.Unitary(np.random.random((8, 8)), 1, 2, 3))
-    c.add(gates.Unitary(np.random.random((8, 8)), 0, 2, 4))
-    c.add(gates.Unitary(np.random.random((2, 2)), 2))
-    c.add(gates.Unitary(np.random.random((2, 2)), 4))
-    c.add(gates.Unitary(np.random.random((8, 8)), 0, 2, 5))
-    _, fig = plot_circuit(c)
+
+    backend = construct_backend("numpy")
+    backend.set_seed(42)
+    np.random.seed(42)
+
+    circuit = Circuit(6)
+    circuit.add(gates.Unitary(random_unitary(8, backend=backend), 1, 2, 3))

```

Comment:
If you want, the `random_unitary` function also accepts `seed` as an argument.

Reply:
good! I set it there too!!